app/controllers/locations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class LocationsController < ApplicationController
+  def create
+    @location = build_location
+
+    if update_location?
+      redirect_to profile_url
+    else
+      @profile = current_user
+
+      render "profiles/show"
```

Comment:
Since `User#location`'s form is on the profile page, is there a better way to render validation errors?